tasks/finger_tapping_task/server_finger_tapping_task.py

```diff
@@ -99,7 +100,7 @@ def _to_client_update_state(self):
 
             # Record state of the game
             if session_index >= 0:
-                self._csv_writer.writerow([time(), json.dumps(data)])
+                self._csv_writer.writerow([time.time(), time.monotonic(), psutil.boot_time(), json.dumps(data)])
```

Comment:
We should label each column in the CSV file in the init function of this class. Simply write the first row as ["time", "monotonic_time", "boot_time", "image_path", "state"]